filterpy/kalman/kalman_filter.py

```diff
@@ -304,49 +304,57 @@ def predict(self, u=0):
         self._P = self._alpha_sq * dot3(self._F, self._P, self._F.T) + self._Q
 
 
-    def batch_filter(self, zs, Rs=None, update_first=False):
+    def batch_filter(self, zs, Rs=None, Fs=None, Hs=None, Qs=None, update_first=False):
         """ Batch processes a sequences of measurements.
-
         **Parameters**
-
         zs : list-like
             list of measurements at each time step `self.dt` Missing
             measurements must be represented by 'None'.
-
         Rs : list-like, optional
             optional list of values to use for the measurement error
             covariance; a value of None in any position will cause the filter
             to use `self.R` for that time step.
-
+        Fs : list-like, optional
+            optional list of values to use for the update matrix;
+            a value of None in any position will cause the filter
+            to use `self.R` for that time step.
+        Hs : list-like, optional
+            optional list of values to use for the measurement matrix;
+            a value of None in any position will cause the filter
+            to use `self.R` for that time step.
+        Qs : list-like, optional
+            optional list of values to use for the process error
+            covariance; a value of None in any position will cause the filter
+            to use `self.R` for that time step.
```

Comment:
I think most of these comments intend to reference the respective parameter instead `self.R`